ui/helpers/utils/tags.ts

```diff
@@ -0,0 +1,41 @@
+// TODO: Remove restricted import
+// eslint-disable-next-line import/no-restricted-paths
+import { getEnvironmentType } from '../../../app/scripts/lib/util';
+import { getIsUnlocked } from '../../ducks/metamask/metamask';
+import {
+  ApprovalsMetaMaskState,
+  getInternalAccounts,
+  getPendingApprovals,
+  getTransactions,
+  selectAllTokensFlat,
+} from '../../selectors';
+import { getMetamaskNotifications } from '../../selectors/metamask-notifications/metamask-notifications';
+import { NftState, selectAllNftsFlat } from '../../selectors/nft';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function getStartupTraceTags(state: any) {
```

Comment:
Nit: A JSDoc block here would be great too

Reply:
Done.